src/components/scenes/courses/CourseManagement/CourseManagement.module.sass

```diff
@@ -0,0 +1,8 @@
+@import '../../../../App.module'
+
+.courseInfo
+  padding: 10px 20px 40px 20px
+
+.courseName
+  margin-bottom: 20px
+
```

Comment:
remove CourseManagement.test.js or complete it

Reply:
deleted :P